docker-compose.yaml

```diff
@@ -13,44 +13,46 @@ services:
   influx:
     image: influxdb:1.7-alpine
     ports:
-      - 8086:8086
+      - "${FLUXDB_PORT}:8086"
     volumes:
       - influx_data:/var/lib/influxdb
       - ./resources/init-fluxdb:/docker-entrypoint-initdb.d
       - /etc/localtime:/etc/localtime:ro
     environment:
-      INFLUXDB_ADMIN_USER: fluxdb-user
-      INFLUXDB_ADMIN_PASSWORD: mysecretpassword
+      INFLUXDB_ADMIN_USER: ${INFLUXDB_ADMIN_USER}
+      INFLUXDB_ADMIN_PASSWORD: ${INFLUXDB_ADMIN_PASSWORD}
       INFLUXDB_HTTP_AUTH_ENABLED: "false"
 
   postgres:
     image: postgres:11-alpine
     ports:
-      - 5432:5432
+      - "${POSTGRES_PORT}:5432"
     volumes:
       - postgres_data:/var/lib/postgresql/data
       - ./resources/init-postgres:/docker-entrypoint-initdb.d
     environment:
-      POSTGRES_DB: thingy-db-red
-      POSTGRES_PASSWORD: mysecretpassword
+      POSTGRES_DB: ${POSTGRES_DBNAME}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD} 
 
   api:
     image: thingy-api-red
     build: ./
     restart: on-failure
     ports:
-      - 8000:8000
+      - ${NODE_PORT}:8000
     volumes:
       - /etc/localtime:/etc/localtime:ro
     depends_on:
       - influx
       - postgres
     environment:
-      NODE_PORT: 8000
-      NODE_ENV: development
+      NODE_PORT: ${NODE_PORT}
+      NODE_ENV: ${NODE_ENV}
       FLUXDB_HOST: influx
+      FLUXDB_PORT: ${FLUXDB_PORT}
       POSTGRES_HOST: postgres
-      MQTT_HOST: thing.mqtt.zone
-      MQTT_PORT: 1893
-      MQTT_USER: red
-      MQTT_PW: 061526c657
+      POSTGRES_PORT: ${POSTGRES_PORT}
+      MQTT_HOST: ${MQTT_HOST}
+      MQTT_PORT: ${MQTT_PORT}
+      MQTT_USER: ${MQTT_USER}
+      MQTT_PW: ${MQTT_PW} 
```

Comment:
@pf15ese good idea to move stuff to `.env.example`. But it would be better to use https://docs.docker.com/compose/compose-file/#env_file to not repeat variables this much time